xml_inscriptions/transcribed/MI.AA.UM.KM.G.21032.xml

```diff
@@ -0,0 +1,126 @@
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#inscribed"> </handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p>relief depicting a standing male figure and seated jackal to the left facing the figure.</p>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               <history><!--summary, origin, provenance go here-->
+                  <summary>
+                     <rs/>  <!-- content can be "modern" or blank -->
+                  </summary>
+                  
+                  <origin>
+                     <date evidence="#archaeology" notBefore="0275" notAfter="0340">late third to
+                        early fourth century CE</date>
+                     <placeName ref="africa.egypt">Egypt, Terenouthis (Kom Abou Billou)</placeName>
+                  </origin>
+                  <provenance type="found" subtype="discovered">
+                     <p>Uncovered during excavations in modern Kom Abou Billou (the modern name of the necropolitan site of the ancient city of Terenouthis) conducted in February-April 1935 by the University of Michigan.</p>
+                  </provenance>
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#SEG_20"/>
+                  <biblScope unit="item">no. 571</biblScope>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include  href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2022-07-24" who="Sophia Papandonatou">added metadata, transcription, translation, commentary</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--facsimile element goes here--><text>
+      <body>
+         <div type="edition">
+         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+            <ab>
+               <lb n="1"/><persName><name key="Ἑρμᾶ Ἑρμᾶς Herma">Ἑρμᾶ<supplied reason="omitted">ς</supplied></name></persName> ἄλυπε 
```

Comment:
Are you giving the inflected form and the nominative in the key? Personally I'd only put the nominative, since that's how we'd probably want to lemmatize these things down the road.